hazelcast/src/main/java/com/hazelcast/internal/partition/impl/MigrationStats.java

```diff
@@ -44,33 +45,83 @@
  */
 public class MigrationStats {
 
+    /**
+     * In {@link TimeUnit#MILLISECONDS}
+     *
+     * @see #getLastRepartitionTime()
+     */
```

Comment:
Nitpick: not really necessary to annotate this; it's an internal class, and all of the information given in the javadoc is shown by the variable's name and `@Probe` annotation

Reply:
I totally missed that, good spot.

Reply:
Just seems like unnecessary noise to me; the method having Javadoc makes sense as it's exposed as part of our API.